test/TwitterAPIExchangeTest.php

```diff
@@ -322,4 +322,23 @@ public function testPut()
         /** If we get this back, then it looks like we can support PUT! :-) **/
         $this->assertContains('UNAUTHORIZED_CLIENT_APPLICATION', $data);
     }
-}
\ No newline at end of file
+
+    public function testDelete()
+    {
+        $params = array();
+
+        // Request create id
+        $url = 'https://ads-api-sandbox.twitter.com/1/accounts';
+        $method = 'POST';
+        $account_data = $this->exchange->request($url, $method, $params);
+        $id = json_decode($account_data)->data->id;
+
+        // Request delete
+        $url = $url.'/'.$id;
+        $method = 'DELETE';
+        $account_data = $this->exchange->request($url, $method, $params);
+        $deleted = json_decode($account_data)->data->deleted;
+
+        $this->assertTrue($deleted);
```

Comment:
This isn't a good test - you need to assert that we actually get valid data back through and not unauthorised.